src/main/java/com/greeenai/greeenai/global/common/constants/SecurityConstants.java

```diff
@@ -1,4 +1,4 @@
-package com.greeenai.greeenai.global.common;
+package com.greeenai.greeenai.global.common.constants;
```

Comment:
다른 package들은 모두 단수형을 쓰고 있어요
통일하면 좋을 것 같습니다.
```suggestion
package com.greeenai.greeenai.global.common.constant;
```

Reply:
constant는 단수형으로 사용했을 때 `상수`라는 의미보다 `일관적인`이라는 형용사의 느낌이 더 있고, 현재 상수들을 관리하고 있는 파일명이 `Constants`로 끝나고 있어서 복수형을 사용했어요. 복수형을 사용하는 건 어떤가요?

Reply:
음..
상수를 사전에서 찾아보면 Constant가 나오기도 하고 
여러 상수를 관리하니까 복수형으로 가자기에는 service나 domain 들도 복수형으로 가야합니다.

오히려 기존 클래스명등을 단수형으로 바꾸는게 더 적절해보입니다.

Reply:
이해했습니다. 그러나 여러 상수들을 관리하는데 단수형을 사용하는게 더 어색한 느낌이 드는 것도 사실입니다. 말씀해주신 대로 일관성을 지키기 위해 아예 복수형으로 통일하는 건 어떨까요?

Reply:
좋습니다
패키지 명은 단수, 상수 클래스명은 복수로 가요~